consensus/vbft/service.go

```diff
@@ -1081,6 +1081,22 @@ func (self *Server) onConsensusMsg(peerIdx uint32, msg ConsensusMsg, msgHash com
 	}
 }
 
+func (self *Server) verifyCrossChainMsg(msg *blockProposalMsg) bool {
+	if msg.Block.CrossChainMsg == nil {
+		return true
+	}
+	root, err := self.chainStore.getCrossStatesRoot(msg.Block.CrossChainMsg.Height)
+	if err != nil {
```

Comment:
call chainstore member function should use self.blockPool interface

Reply:
fixed